src/components/modals/loginModal.js

```diff
@@ -1,20 +1,35 @@
-export const LoginModal = ({
-  showModalLogin,
-  setShowModalLogin,
-  showModalSignUp,
-  setShowModalSignUp,
-}) => {
+import React, { useContext } from "react";
+import { ModalsContext } from "../../Contexts/modalsContext";
+import { useForm } from "react-hook-form";
+
+export const LoginModal = () => {
+  const {
+    showSignInModal,
+    setShowSignInModal,
+    showSignUpModal,
+    setShowSignUpModal,
```

Comment:
Thought about that and what do you think about passing string like `SIGN_UP` or `SIGN_IN` into state called `modal`, and based on that string show appropriate content?